modules/installation-guide/partials/proc_configuring-workspace-exposure-strategies-using-an-operator.adoc

```diff
@@ -22,9 +23,10 @@ To activate changes done to CheCluster YAML file, do one of the following:
 ----
 $ {prod-cli} server:deploy  --installer=operator --platform=... --che-operator-cr-patch-yaml=patch.yaml
```

Comment:
What's the expected value for `--platform=...`?
```suggestion
$ {prod-cli} server:deploy  --installer=operator --platform=... \
  --che-operator-cr-patch-yaml=patch.yaml
```

Reply:
Will be solved